send_multiple_messages.py

```diff
@@ -0,0 +1,41 @@
+import argparse
+import sys
+import threading
+
+from client import send_data
+
+
+def send_messages(server_ip, server_port, messages):
```

Comment:
This is not implemented, delete this file?

Reply:
The implementation on `client.send_data` has changed, but its public signature and what it does have remained the same, so this file is unaffected.